packages/otelbin/src/app/s/[id]/metadataUtils.test.ts

```diff
@@ -50,13 +50,19 @@ describe("calcScale", () => {
 		const nodes: Node[] = [
 			{ type: "parentNodeType", data: { height: 100 }, position: { x: 0, y: 0 }, id: "1" },
 			{ type: "parentNodeType", data: { height: 200 }, position: { x: 0, y: 0 }, id: "2" },
-			{ type: "processorsNode", data: { height: 100 }, position: { x: 0, y: 0 }, id: "3" },
-			{ type: "processorsNode", data: { height: 100 }, position: { x: 0, y: 0 }, id: "4" },
+			{ type: "processorsNode", parentNode: "metrics", data: { height: 100 }, position: { x: 0, y: 0 }, id: "3" },
```

Comment:
**Suggestion:** Add a few more test cases for the other permutations listed in the issue

Reply:
Test cases added and pushed